app/src/main/kotlin/com/jshvarts/daggerandroidsampleapp/com/jshvarts/daggerandroidmvp/lobby/LobbyActivity.kt

```diff
@@ -1,4 +1,4 @@
-package com.jshvarts.daggerandroidsampleapp.lobby
+package com.jshvarts.daggerandroidsampleapp.com.jshvarts.daggerandroidmvp.lobby
```

Comment:
There is some funkiness going on with packages here. The new package is wrong

Reply:
it was suggested by Android studio, I could revert it